src/main/resources/assets/spruceui/lang/en_ud.json

```diff
@@ -0,0 +1,12 @@
+{
+ "spruceui.reset": "ʇesǝᴚ",
+ "spruceui.gui.unbind": "dnᴉbn∩",
+ "spruceui.narrator.separator": "roʇaɹɐdǝs %s",
+ "spruceui.narration.toggle_switch": "%s etɐts htiw ,%s :ɥɔʇᴉʍS ǝןgᵷo⟘",
+ "spruceui.options.generic": "%s :%s",
```

Comment:
IIrc, variables like that can be re-ordered, I don't remember the format though, might be a thing to investigate for a proper upside-down translation!

Reply:
Good call. I've looked up the format and implemented it.